.github/workflows/arm-runner.yml

```diff
@@ -113,29 +112,44 @@ jobs:
             echo ::set-output name=LOCAL_BRANCH::${GITHUB_REF/refs\/tags\//}
             echo ::set-output name=RELEASE_NAME::${GITHUB_REF/refs\/tags\//}
             echo ::set-output name=IMAGE_NAME_SUFFIX::${GITHUB_REF/refs\/tags\//}-${{ matrix.target }}
+            echo ::set-output name=CLONE_RELEASE::yes
           elif [ ${GITHUB_REF/refs\/heads\//} = "releng" ]; then
             echo ::set-output name=LOCAL_BRANCH::releng
             echo ::set-output name=RELEASE_NAME::releng
             echo ::set-output name=IMAGE_NAME_SUFFIX::releng-${{ matrix.target }}
-          else
+            echo ::set-output name=CLONE_RELEASE::yes
+          elif [ ${GITHUB_REF/refs\/heads\//} != "${GITHUB_REF}" ]; then
             echo ::set-output name=LOCAL_BRANCH::${GITHUB_REF/refs\/heads\//}
+            echo ::set-output name=RELEASE_NAME::${GITHUB_REF/refs\/heads\//}
+            echo ::set-output name=IMAGE_NAME_SUFFIX::${GITHUB_REF/refs\/heads\//}-${{ matrix.target }}
+            echo ::set-output name=CLONE_RELEASE::yes
+          else
+            echo ::set-output name=LOCAL_BRANCH::${GITHUB_REF}
+            echo ::set-output name=IMAGE_NAME_SUFFIX::${GITHUB_REF//\//-}-${{ matrix.target }}
+            echo ::set-output name=CLONE_RELEASE::no
           fi
 
       - name: Create a release image in chroot environment
         uses: pguyot/arm-runner-action@v2
         id: arm_runner_install
         with:
-          image_additional_mb: 4608
+          image_additional_mb: 4096
           base_image: ${{ matrix.base_image }}
           cpu: ${{ matrix.cpu }}
           cpu_info: ${{ matrix.cpu_info }}
           optimize_image: yes
+          copy_repository_path: /home/pi/pynab
           commands: |
             sudo apt-get update -y --allow-releaseinfo-change
             echo GITHUB_BRANCH=${{ steps.image_and_branch_name.outputs.LOCAL_BRANCH }}
             echo GITHUB_REPOSITORY=${{ github.repository }}
+            if [ ${{ steps.image_and_branch_name.outputs.CLONE_RELEASE }} = "yes" ]; then
+                sudo rm -rf /home/pi/pynab
+            else
+                sudo useradd pi || true
```

Comment:
We might want to avoid creating user pi on dietpi distributions.

Reply:
Indeed. Or rather manage to delete it afterwards...:
On DietPi, `setup` now deletes `pi` user after moving pynab directory over to `dietpi` user.